appchallenge/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>MCC</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="details.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+	<div data-role="header">
+	<h1>MCC Sage Bistro</h1>
+	<img src="Sage-Bistro-Logo-2017-transparenent-small.png" id="logo">
+	</div>
+	<div data-role="header">
+	<a href="maps.html" class="ui-btn-right"> Find Us</a>
+	<a href="survey.html" class="ui-btn"> Take Survey</a>
+	<a href="reservation.html" class="ui-btn"> Reservation</a>
+	<a href="menu.html" class="ui-btn"> Menu</a>
+	</div>
+	<div role="main" class="ui-content">
+	<p>Welcome to the main page for the Metro Community College Student Bistro, a student-run restaurant sponsored by the 
+	Omaha Metro Community College. This page is a sort of hub. From here you can access information about our location, a survey for
+	you to provide us feedback, a page where you can make a reservation and our full menu. There are buttons above and links below. 
+	We hope that you find the website easy to navigate and that you choose to dine with us soon.</p>
+	</div>
+	<div role="main" class="ui-content">
+	<ul data-role="listview" data-insert="true">
+		<li> <a href="maps.html">Find Us</a></li>
+		<li><a href="survey.html">Take The Survey</a></li>
+		<li><a href="reservation.html">Make A Reservation</a></li>
+		<li><a href="menu.html">Check The Menu</a></li>
+	</ul>
+	<h2>Gallery:</h2>
+	<img src="MCC_Sage_Resources/sb_dining_room_01.jpg">
+	<img src="MCC_Sage_Resources/sb_dining_room_02.jpg">
+	<img src="MCC_Sage_Resources/sb_dining-room_03.jpg">
+	<img src="MCC_Sage_Resources/sb_wine_room.jpg">
+	</div>
```

Comment:
This was not required - but I like it! I do agree with one of your users that making all images the same size would help polish it up.